src/Datasource/EntityTrait.php

```diff
@@ -527,21 +527,11 @@ public function has(array|string $field): bool
      *
      * @param string $field The field to check.
      * @return bool
+     * @deprecated 5.3.0 Use hasValue() instead.
```

Comment:
Shouldn't we also have a runtime deprecation?

Reply:
If someone wants to add it, be my guest. I already overdid myself the last 3 days :)

Reply:
Please don't leave PRs for others to complete :)
All that's needed is additional 1 line of code.